torchgeo/datasets/__init__.py

```diff
@@ -115,6 +116,7 @@
     "ChesapeakeVA",
     "ChesapeakeWV",
     "ChesapeakeCVPR",
+    "CloudCoverDetection",
```

Comment:
Can you also add a section to `docs/api/datasets.rst`? Should go under the non-geospatial datasets section.